games-sports/speed-dreams/patches/speed_dreams-2.2.2.patchset

```diff
@@ -0,0 +1,209 @@
++	 ~/Library/Frameworks/plib.framework
++	 /Library/Frameworks/plib.framework
++	 /System/Library/Frameworks/plib.framework # Tiger
+ 
+ 	/usr /usr/local
++	/system
+   DOC "Location of plib")
+ ENDIF(NOT APPLE OR OPTION_USE_MACPORTS)
+ 
+diff --git a/src/libs/tgfclient/CMakeLists.txt b/src/libs/tgfclient/CMakeLists.txt
+index 77b7a1d..1cabf1f 100644
+--- a/src/libs/tgfclient/CMakeLists.txt
++++ b/src/libs/tgfclient/CMakeLists.txt
+@@ -111,12 +111,12 @@ IF(WIN32)
+     TARGET_LINK_LIBRARIES(tgfclient winmm)
+ ENDIF(WIN32)
+ 
+-IF(UNIX)
++IF(UNIX AND NOT HAIKU)
+     ADD_X11_LIBRARY(tgfclient)
+     IF(OPTION_XRANDR)
+         ADD_XRANDR_LIBRARY(tgfclient)
+     ENDIF(OPTION_XRANDR)
+-ENDIF(UNIX)
++ENDIF(UNIX AND NOT HAIKU)
+ 
+ IF(UNIX)
+     SD_INSTALL_FILES(LIB lib TARGETS tgfclient)
+-- 
+2.21.0
+
+
+From 81b490317d12195bc444cbc814ac0d7024a46278 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 18 May 2019 14:01:33 +0200
+Subject: Disable code unsupported on Haiku.
+
+
+diff --git a/src/libs/tgf/linuxspec.cpp b/src/libs/tgf/linuxspec.cpp
+index 12d663e..b6ed88f 100644
+--- a/src/libs/tgf/linuxspec.cpp
++++ b/src/libs/tgf/linuxspec.cpp
+@@ -692,7 +692,7 @@ unsigned linuxGetNumberOfCPUs()
+ * Remarks
+ *    
+ */
+-#if !defined(USE_MACPORTS)
++#if !defined(USE_MACPORTS) || !defined(__HAIKU__)
+ std::string cpuSet2String(const cpu_set_t* pCPUSet)
+ {
+ 	std::ostringstream ossCPUSet;
+-- 
+2.21.0
+
+
+From 0992532c46b5dfadb0544cfb90f5cbce51009832 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 18 May 2019 14:07:05 +0200
+Subject: Use namespace std
+
+
+diff --git a/src/modules/simu/simuv2.1/simu.cpp b/src/modules/simu/simuv2.1/simu.cpp
+index cac11d7..9c4377f 100644
+--- a/src/modules/simu/simuv2.1/simu.cpp
++++ b/src/modules/simu/simuv2.1/simu.cpp
+@@ -28,6 +28,7 @@
+ 
+ #include "sim.h"
+ 
++using namespace std;
+ 
+ tCar *SimCarTable = 0;
+ 
+-- 
+2.21.0
+
+
+From 41740871050c1a173e8568e3a9a6498752a413e3 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 18 May 2019 14:09:58 +0200
+Subject: Do not link against dl
+
+
+diff --git a/src/tools/nfsperf/CMakeLists.txt b/src/tools/nfsperf/CMakeLists.txt
+index 0a2a6b6..d9fc80e 100644
+--- a/src/tools/nfsperf/CMakeLists.txt
++++ b/src/tools/nfsperf/CMakeLists.txt
+@@ -16,9 +16,9 @@ SD_ADD_EXECUTABLE(${_TARGET_NAME} ${NFSPERF_SOURCES})
+ 
+ ADD_SDLIB_LIBRARY(${_TARGET_NAME} portability tgf txml) # txml ignored if OPTION_3RDPARTY_EXPAT
+ 
+-IF(UNIX)
++IF(UNIX AND NOT HAIKU)
+     TARGET_LINK_LIBRARIES(${_TARGET_NAME} dl)
+-ENDIF(UNIX)
++ENDIF(UNIX AND NOT HAIKU)
+ 
+ IF(UNIX)
+     SD_INSTALL_FILES(MAN man6 PREFIX ${SOURCE_DIR}/doc/man FILES ${_TARGET_NAME}.6)
+@@ -36,4 +36,4 @@ IF((APPLE) AND ("${CMAKE_INSTALL_PREFIX}" MATCHES "\\.app$"))
+    INCLUDE(BundleUtilities)
+    SET(BU_CHMOD_BUNDLE_ITEMS ON)
+    FIXUP_BUNDLE(\"${APPS}\" \"${PlugIns}\" \"${DIRS}\")")
+-endif()
+\ No newline at end of file
++endif()
```

Comment:
you could remove this chunk

Reply:
@korli this is generated by extracting the patchset?

Reply:
not exactly, you edited with Pe which added this newline at the end. this is just noise in this patch and won't be upstreamed.

Reply:
Ok, need to investigate that how to remove without braking the patchset

Reply:
Think this should do it